### PR TITLE
Feature/home ecom 8

### PR DESCRIPTION
**Pull request by somaya-mahmoud**
feat : add home screen design .

### CODE REVIEW

File: assets/images/deliver_to.png
Comment (Marco-Nagy) [inline assets/images/deliver_to.png:1]: this is Text not image 
please refactor it to text with the same font and size and color

File: assets/images/place.png
Comment (Marco-Nagy) [inline assets/images/place.png:1]: this is Text not image 
please refactor it to text with the same font and size and color

File: lib/features/home/presentation/screens/home_screen.dart [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                  icon: Image.asset(
+                    'assets/images/home.png',
+                    width: 24.w,
+                    height: 24.h,
+                  ),
+                  label: 'Home',
+                ),
+                BottomNavigationBarItem(
+                  icon: Image.asset(
+                    'assets/images/category.png',
```
Comment (Marco-Nagy) [inline lib/features/home/presentation/screens/home_screen.dart]: please use AppImages class to store images path as static const

File: lib/features/home/presentation/screens/home_screen.dart [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                  icon: Image.asset(
+                    'assets/images/shopping_cart.png',
+                    width: 24.w,
+                    height: 24.h,
+                  ),
+                  label: 'Cart',
+                ),
+                BottomNavigationBarItem(
+                  icon: Image.asset(
+                    'assets/images/person.png',
```
Comment (Marco-Nagy) [inline lib/features/home/presentation/screens/home_screen.dart]: please use AppImages class to store images path as static const

File: lib/features/home/presentation/tabs/cart_tab.dart [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  const CartTab({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      color: Colors.white,
+      child: Center(
+          child: Text(
+            "Cart Tab",
+            style: TextStyle(
```
Comment (Marco-Nagy) [inline lib/features/home/presentation/tabs/cart_tab.dart]: please use MyFonts class and use font style

File: lib/features/home/presentation/tabs/home_tab.dart [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+  const HomeTab({super.key});
+  @override
+  Widget build(BuildContext context) {
+    final searchController = TextEditingController();
+    return Scaffold(
+      appBar: AppBar(
+        title: Row(
+          children: [
+            Image.asset(
+              'assets/images/logo.png',
```
Comment (Marco-Nagy) [inline lib/features/home/presentation/tabs/home_tab.dart:19]: please use AppImages class to store images path as static const

File: lib/features/home/presentation/tabs/profile_tab.dart [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  const ProfileTab({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      color: Colors.white,
+      child: Center(
+          child: Text(
+            "Profile Tab",
+            style: TextStyle(
```
Comment (Marco-Nagy) [inline lib/features/home/presentation/tabs/profile_tab.dart]: Please use Myfonts  and refactor it with app font styles

**Review by Marco-Nagy — CHANGES_REQUESTED** [review]
please refactor MyFonts Class with flowery app text style
 FontWeigt
 FontSize
 add images paths to AppImages Class and use it as static const

**Review by Marco-Nagy — APPROVED** [review]
عااااش اوى الله ينور يا هندسة 🙌

**Comment by somaya-mahmoud** [comment]
fix : use app images class to store path of images , and Use MyFonts class with app font style.

**Comment by somaya-mahmoud** [comment]
fix : use app images to store images path and use MyFonts class with flower app font style.

**Review by Marco-Nagy — APPROVED** [review]

**Review by Marco-Nagy — APPROVED** [review]

**Review by haidyhesham95 — APPROVED** [review]

**Review by YasmeenGad — APPROVED** [review]

**Pull request merged**